extension/module/module.cpp

```diff
@@ -302,5 +302,15 @@ runtime::Error Module::set_output(
       output_tensor.mutable_data_ptr(), output_tensor.nbytes(), output_index);
 }
 
+ET_NODISCARD inline runtime::Result<Method*> Module::get_method(
```

Comment:
Why do we need to expose the Method? Can we make some ivars protected instead and let subclasses work with them directly, like this class itself?
Also, there's no get_-prefix naming convention for this class, it should be named just "method()", if really needed.

Reply:
@shoumikhin that should my fault i misread the scope of method list as private so I asked to put the function here. 
Since it is protected, making it as a child’s private function should be a better choice.

Reply:
Now worries, you should be able to access `Method` via `methods_.at(method_name).method` in subclasses indeed.